gcloud/_helpers.py

```diff
@@ -48,6 +51,7 @@
     (?P<nanos>\d{1,9})                       # nanoseconds, maybe truncated
     Z                                        # Zulu
 """, re.VERBOSE)
+DEFAULT_CONFIGURATION_PATH = '~/.config/gcloud/configurations/config_default'
```

Comment:
Windows it appears puts it in this location
`C:\Users\<USER>\AppData\Roaming\gcloud\configurations\config_default`. So for windows it will fail this check and move on to another check.